erpnext/support/report/issue_summary/issue_summary.js

```diff
@@ -42,6 +42,7 @@ frappe.query_reports["Issue Summary"] = {
 				"",
 				{label: __('Open'), value: 'Open'},
 				{label: __('Replied'), value: 'Replied'},
+				{label: __('Hold'), value: 'Hold'},
```

Comment:
Can we rename this as "On Hold"? "On Hold Issues" makes more sense than "Hold Issues"

Reply:
'On Hold' seems good.